frank-flow/src/frontend/src/app/flow/flow.component.ts

```diff
@@ -9,6 +16,65 @@ import { NodeService } from './node/node.service';
 export class FlowComponent {
   nodes = [];
   connections = [];
+  private panZoomConfigOptions: PanZoomConfigOptions = {
+    zoomLevels: 10,
+    zoomStepDuration: 0.2,
+    freeMouseWheelFactor: 0.01,
+    zoomToFitZoomLevelFactor: 0.5,
+    dragMouseButton: 'left',
+    zoomOnDoubleClick: false,
+  };
+  panzoomConfig: PanZoomConfig = new PanZoomConfig(this.panZoomConfigOptions);
 
-  constructor(private nodeService: NodeService) {}
+  @ViewChild('nodeContainer', { read: ElementRef })
+  nodeContainerRef!: ElementRef;
+  private offset = 500;
+
+  constructor(private renderer: Renderer2, private library: FaIconLibrary) {
+    this.library.addIcons(faArrowDown, faArrowUp, faArrowRight, faArrowLeft);
+  }
+
+  decreaseRight(): void {
+    const el = this.nodeContainerRef.nativeElement.getElementsByClassName(
```

Comment:
It just occurred to me that canvas is a class name, this should be changed to id, because canvas isn't something that gets used repeatedly. The canvas should always be the same element: hence, use id.